xbmc/guilib/GUIAudioManager.cpp

```diff
@@ -86,18 +93,17 @@ void CGUIAudioManager::DeInitialize()
 void CGUIAudioManager::Stop()
 {
   CSingleLock lock(m_cs);
-  for (auto& it : m_windowSoundMap)
+  for (const auto& it : m_windowSoundMap)
```

Comment:
as we don't get an iterator maybe rename `it` to `soundMap` (or `windowSoundMap`)

Reply:
I think `windowSound` would be very fitting, I'll change it to that.